src/Umbraco.Tests/Clr/ReflectionUtilitiesTests.cs

```diff
@@ -307,13 +307,6 @@ public void SetterCanCastUnsafeValue()
 
             setterInt4(object4, 42);
             Assert.AreEqual(42, object4.IntValue);
-
-            // converting works
-            setterInt4(object4, 42.0);
```

Comment:
that line should work and not crash - is it crashing?

Reply:
This is just for the record here, for completeness: yes, it crashes with an ExecutionEngineException. More details in the issue U4-11390.